source/ns_cmdline.c

```diff
@@ -231,8 +240,14 @@ int history_command(int argc, char *argv[]);
 
 void default_cmd_response_out(const char *fmt, va_list ap)
 {
+#ifdef MBED_CONF_EXTERNAL_MBED_TRACE_ENABLED
```

Comment:
this would be breaking change. original functionality should be active by default and unified logging related stuff behind some flag, otherwise all linux builds would fail.

Reply:
Well obviously it will be breaking change as it is dependent on new logging feature which is not yet part of mbed-is master. I gave you this change to show how we can have all prints synchronised. I hope you can verify this with unified logging PR and let me know if it covers all cases of client cli or not. Will revert it after 2 days as it is not needed in my mind this PR

Reply:
Its good to know how syncing could be implemented using unified logging. We could add examples which do this in application level.